servers.go

```diff
@@ -24,6 +24,13 @@ type Server struct {
 	Platform   string `json:"platform"`
 }
 
+//Are used to create user(s) on new machine
+type User struct {
+	Username     string   `json:"username"`
+	PublicKey    []string `json:"sshkeys"`
```

Comment:
plural för consistency?

```suggestion
	PublicKeys    []string `json:"sshkeys"`
```